### ISSUE TITLE
What's a strange version name?

### ISSUE THREAD

**Issue by pengcao1** [react-native-community/jsc-android-buildscripts#127]
Hi guys:
your know,  lot of you guys upgrade react-native to 0.60 or above, there are many many `libjsc.so` crashes were reported from google play.  and many friendly developers suggest me to change the runtime engine from default to v8 when I try to fix this issue. 
I just want to change the default engine to `js-android`, but, when I do this, the `js-android` was so strange!!, 
**could anybody help to describe the rule of the version number?**

-----
**Version number from [*js-android-npm*](https://www.npmjs.com/package/jsc-android)
![image](https://user-images.githubusercontent.com/40621240/68585852-5552ab80-04be-11ea-87df-b393ef4dda3f.png)

**Comment by tobi512** [comment]
Hi @pengcao1 
it's called `jsc-android` and _not_ `js-android`, be careful about that (JSC = JavaScriptCore).

The version names are based on the SVN revision of the repository (https://svn.webkit.org/repository/webkit/), which is basically the commit-count if you're not into SVN (so higher is newer). I'm not sure what the last two digits in the versioning mean, could be some kind of minor/patch numbers, but that's actually not really important since you have the revision itself.

Btw: If you wanna switch your engine from JSC (currently React Native already comes with JSC `245459.0.0` by default) to v8 (or maybe even Hermes on Android) you can basically do so without even touching the JSC, since the engines are completely independent...

**Comment by newyankeecodeshop** [comment]
@tobi512 I think there is an issue with the NPM tags though, because when I run `yarn outdated`, it shows `241213.1.0` as the latest. Since RN 0.60 ships with `245459.0.0`, that version should be marked as the latest. Also the "Releases" tab in this repo doesn't have any of the `24xxxx` releases at all.

**Comment by tobi512** [comment]
@newyankeecodeshop Yes that's indeed strange, `245459.0.0` is still tagged as `next` on https://www.npmjs.com/package/jsc-android and `stock-react-native` points to `174650.0.2` which seems ancient. 
Anyone here knows why that is the case?
